openmdao/core/tests/test_remote_vois.py

```diff
@@ -73,6 +71,46 @@ def test_remote_voi(self):
         assert_near_equal(J['par.G2.c', 'par.G1.x'], np.array([[0.0]]), 1e-6)
         assert_near_equal(J['par.G2.c', 'par.G2.x'], np.array([[1.0]]), 1e-6)
 
+    def test_check_totals_matfree_w_remote(self):
+        # this test used to hang before the fix for issue # 2884
```

Comment:
suggest including the gist of issue here to facilitate understanding of the test, e.g.
```
# check_totals was hanging in some cases when all variables of interest didn't exist on all procs. (Issue #2884)
```

Reply:
I added a link to the issue.